.gitignore

```diff
@@ -1,3 +1,60 @@
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided this notice is
+# preserved.  This file is offered as-is, without any warranty.
+# Names of contributors must not be used to endorse or promote products
```

Comment:
oh well, it was fine english-wise, but I don't like additional custom licenses, the main gpl-2-ish license of ettercap should be fine for a gitignore file too :)
(and as a Debian maintainer I really don't like having to patch my copyright file just for it)